packages/entities/entities-shared/src/composables/useErrors.ts

```diff
@@ -70,7 +70,99 @@ export default function useErrors() {
     return error.message || t('errors.unexpected')
   }
 
+  /**
+ * Receive an error and return the error fields from the payload
+ * @param {string|Error|object} error - error string
+ * @returns {Object} Object containing message and fields
+ */
+  const getErrorFieldsFromError = (error: any): { messages: string[]; fields: Array<{ field: string; message: string }> } => {
+    if (!error) {
+      return {
+        messages: [t('errors.unexpected')],
+        fields: [],
+      }
+    }
+
+    if (error?.response?.status === 401) {
+      console.warn('getErrorFieldsFromError: User auth token is expired or missing, returning empty messages.')
```

Comment:
question: what's the purpose of this?

Reply:
Checks if the error response is 401; if so, handle it appropriately without returning an error message.

Reply:
But why is there a `console.warn`?

Reply:
removed the console.warn